### PR TITLE
Welford consistency

### PR DESCRIPTION
**Pull request by csarofeen**
Was having issues on another branch where welford output wasn't consistent for scheduling. Sometimes the variance would be marked with compute/produce at, sometimes average. I went through and made sure welford is always of the order average, variance, then count. Making sure output/input is always average.

### CODE REVIEW

**Comment by csarofeen** [comment]
Looks like I have some issues even after this. Still need to investigate further.

**Comment by csarofeen** [comment]
@shmsong this is ready for review. Thanks.

File: test/cpp/jit/test_gpu.cpp [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#if defined(USE_CUDA)
+// #if defined(USE_CUDA)
```
Comment (shmsong) [inline test/cpp/jit/test_gpu.cpp]: Please uncomment this before merging.

**Review by shmsong — APPROVED** [review]
LGTM! Thanks for the thorough fixing of input and output ordering 👍

**Pull request merged**